pyproject.toml

```diff
@@ -1,3 +1,120 @@
 [build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = ["setuptools>=64", "setuptools_scm[toml]>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "atip"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+description = "Accelerator Toolbox Interface for Pytac"
+dependencies = [
+    "numpy",
+    "scipy",
+    "pytac>=0.3.0",
+    "accelerator-toolbox>=0.2.0",
+    "cothread",
+    "softioc",
+]
+
+dynamic = ["version"]
+license.file = "LICENSE"
+readme = "README.md"
+requires-python = ">=3.10"
+
+[project.optional-dependencies]
+dev = [
+    "copier",
+    "mypy",
+    "pipdeptree",
+    "pre-commit",
+    "pytest",
+    "pytest-cov",
+    "testfixtures",
+    "ruff",
+    "tox-direct",
+    "types-mock",
+]
+
+[project.scripts]
+atip = "atip.__main__:main"
+virtac = "virtac.__main__:main"
+
+[project.urls]
+GitHub = "https://github.com/DiamondLightSource/atip"
+
+[[project.authors]] # Further authors may be added by duplicating this section
+email = "tobyn.nicholls@diamond.ac.uk"
+name = "Tobyn Nicholls"
+
+[[project.authors]] # Further authors may be added by duplicating this section
+email = "philip.smith@diamond.ac.uk"
+name = "Phil Smith"
+
+[tool.setuptools_scm]
+version_file = "src/atip/_version.py"
+
```

Comment:
I think you need to configure package data? Something like:
```
[tool.setuptools.package-data]
atip = ['rings']
virtac = ['data']
```
But I can't remember whether you need wildcards etc. specified. You can always build the package and find out for yourself.

This should go immediately under the version_file thing.

Reply:
I.e. equivalent to
```
include atip/rings/*.m*
include virtac/data/*.csv
```
that used to exist in MANIFEST.in

Reply:
Apparently in the most recent versions of setuptools, as long as the file is included in git with setuptools_scm, there is no need to add it manually to package_data, or set include-package-data to True.
https://setuptools.pypa.io/en/latest/userguide/datafiles.html#configuration-options